src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.DataTypeFlags.FLAG_APPLICANT;
+import static seedu.address.commons.core.DataTypeFlags.FLAG_INTERVIEW;
+import static seedu.address.commons.core.DataTypeFlags.FLAG_POSITION;
+
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.applicant.ListApplicantCommand;
+import seedu.address.logic.commands.interview.ListInterviewCommand;
+import seedu.address.logic.commands.position.ListPositionCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+
+
+
+public class ListCommandParser implements Parser<ListCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddApplicantCommand,
```

Comment:
This JavaDocs comment seems to be about `AddApplicantCommand`, do change it for this command accordingly.